packages/components/src/components/File.tsx

```diff
@@ -2,21 +2,24 @@ import { useState } from 'react'
 import { FileKey, UrlKey } from '../lib/key.js'
 import Breadcrumb from './Breadcrumb.js'
 import Layout from './Layout.js'
-import Viewer from './Viewer.js'
+import Viewer, { ViewerConfig } from './viewers/Viewer.js'
+
+export type FileConfig = ViewerConfig
 
 interface FileProps {
-  parsedKey: UrlKey | FileKey
+  parsedKey: UrlKey | FileKey,
```

Comment:
nit: interfaces don't require trailing commas

Reply:
indeed! my bad. note that we sometimes use a `;` and sometimes nothing (and sometimes a `,`) and the three versions are accepted by eslint. I haven't found a rule in https://typescript-eslint.io/rules/, so... not sure what we can do to improve that.

Reply:
(I'm removing the commas, anyway)